src/Main.java

```diff
@@ -17,6 +17,51 @@ public static void main(String[] args) {
 
     }
 
+    static void searchBooks() {
```

Comment:
```suggestion
    static void searchBooks(String input) {
```
olarak parametre vermen daha doğru